source/less/blocks/main-container.less

```
@@ -15,23 +22,26 @@
   width: 1200px;
```

```suggestion
  width: 100%;
```
Псевдоэлемент занимает какую-то часть от родительского. Ты задала 1200px, и если окно уменьшить, то он начинает толкать край окна. Если задать в процентах, то всегда будет в рамках родительского блока